src/game/rtsutils.h

```diff
@@ -89,6 +89,17 @@ template<int a, int b, int c, int d> struct FourCC
 #endif
 };
 
+inline uint32_t FourCC_By_String(const char *s)
+{
+    if (strnlen(s, 5) > 4)
+        return 0;
+#ifdef SYSTEM_LITTLE_ENDIAN
+    return (s[3] << 24) | (s[2] << 16) | (s[1] << 8) | s[0];
+#else
+    return (s[0] << 24) | (s[1] << 16) | (s[2] << 8) | s[3];
```

Comment:
This means on Big Endian system, compression tool user would need to supply "\0RAE" string to select this option. Is that what you want?

Reply:
On second thought, I could be mistaken. Best to check in debugger.